README.md

```diff
@@ -1,24 +1,13 @@
-# README
+# Todo app
```

Comment:
Could you please add steps for setting up the database?

Reply:
Done